app/src/main/java/com/google/android/soundchecker/BaseFilePlayerActivity.kt

```diff
@@ -318,4 +323,13 @@ open class BaseFilePlayerActivity : ComponentActivity(), OnAudioFocusChangeListe
             }
         }
     }
+
+    inner class FilePlaybackErrorCallback : AudioErrorCallback() {
+        override fun onError(error: Int, msg: String) {
+            if (error != AudioErrorCallback.SUCCESS) {
+                Log.e(mTag, "Stop playback when receiving error=$error, msg=$msg")
```

Comment:
Will the user see this error?
When writing framework code, logging the error and returning an error ode is enough.
But when writing test apps, if we get an unexpected error from a bug, then we should alert the user.

Reply:
Added a text to show error message